test/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests/Abstractions/IExampleTests.cs

```diff
@@ -5,6 +5,7 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
 using Newtonsoft.Json.Serialization;
+using System;
 
 namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests.Abstractions
 {

```

Comment:
Can you please change the class name from `IExampleTests` to `OpenApiExampleTests`?